Theme/Theme/Assets.xcassets/Colors/CardView/CardViewBackground.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "display-p3",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "1.000",
-          "green" : "1.000",
-          "red" : "1.000"
+          "blue" : "0.984",
+          "green" : "0.980",
+          "red" : "0.976"
```

Comment:
We offer expandable/collapsible sections labeled as "Completed." These sections come with their own background and border stroke colors for both dark and light modes. The designer provided these colors in [Figma designs](https://www.figma.com/file/iZ56YMjbRMShCCDxqrqRrR/Open-edX-Mobile-App-All-Screens-v2.0?type=design&node-id=6703-32751&mode=design&t=wodLeZoiQsg4uBRt-4), which closely resemble our existing color palette. Instead of introducing a new set of colors, I made slight adjustments to the existing palette to ensure its versatility.

For instance, in light mode, the background color of the "Completed" section is now `CCD4E0`, whereas it was previously `FFFFFF` (white). This change was made to address the issue of indistinguishability between the screen background and the section background, using the specified Figma color for clarity.

| Light Mode                                                | Dark Mode                                               |
|--------------------------------------------------------|--------------------------------------------------------|
| ![Simulator Screenshot - iPhone 15 - 2024-01-24 at 20 42 17](https://github.com/openedx/openedx-app-ios/assets/11990137/e7876b7d-1693-47c1-a3b2-9d84ce5de2d8) | ![Simulator Screenshot - iPhone 15 - 2024-01-24 at 20 47 55](https://github.com/openedx/openedx-app-ios/assets/11990137/c9b51c33-29e3-425b-af8a-490abc5e291e) |

Reply:
For safer side, I introduced separate colors for calendar completed section background and stroke colors in light and dark mode.